battle/magitek_upgrade.py

```diff
@@ -0,0 +1,101 @@
+from memory.space import Bank, START_ADDRESS_SNES, Reserve, Write, Read
+import instruction.asm as asm
+
+import data.event_bit as event_bit
+import objectives
+
+class _MagitekUpgrade:
+    '''Set the Magitek menu in battle to match the Magitek Upgrade objective result.'''
+    def __init__(self):
+        # Write our 2 magitek tables
+        # We're moving them from C1/910C - C1/911B
+        # Default: Match Regular character's default
+        src = [
+            0x00, 0x01, #FIRE_BEAM, BOLT_BEAM, 
+            0x02, 0xFF, #ICE_BEAM, <empty>, 
+            0x04, 0xFF, #HEAL_FORCE, <empty>, 
+            0xFF, 0xFF  #<empty>, <empty>
+        ]
+        space = Write(Bank.F0, src, "magitek default table")
+        magitek_default_table_addr = space.start_address
+
+        # Upgraded: Match Terra's options
+        src = [
+            0x00, 0x01, #FIRE_BEAM, BOLT_BEAM,
+            0x02, 0x03, #ICE_BEAM, BIO_BLAST, 
+            0x04, 0x05, #HEAL_FORCE, CONFUSER, 
+            0x06, 0x07  #X_FER, TEKMISSILE
+        ]
+        space = Write(Bank.F0, src, "magitek upgraded table")
+        magitek_upgraded_table_addr = space.start_address

```

Comment:
This can be `space.start_address_snes`, 

start_address_snes simply appends START_ADDRESS_SNES as you are doing below

Reply:
Thanks, changed.